### PR TITLE
Ready for production

### PR DESCRIPTION
**Pull request by mlevit**

### CODE REVIEW

**Comment by mlevit** [comment]
@jihygk can you review the new KMS pytest added. Need to implement as many tests as possible but need to understand your setup a bit more.

File: auto_remediate/test/test_securityhub_kms.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    def test_invalid_invalid_kms_cmk(self, sh):
+        """Tests invalid KMS CMK
+        
+        Arguments:
+            sh {SecurityHubRules} -- Instance of class SecurityHubRules
+        """
+        # test cmk_backing_key_rotation_enabled
+        response = sh.cmk_backing_key_rotation_enabled(
+            "e85f5843-1111-4bcb-b711-7e17fa181804"
```
Comment (jihygk) [inline auto_remediate/test/test_securityhub_kms.py]: Don't understand where the magic number came from? Is this always returned by moto?

File: auto_remediate/test/test_securityhub_kms.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        )
+        assert not response
+
+
+class TestSecurityHubStatic:
```
Comment (jihygk) [inline auto_remediate/test/test_securityhub_kms.py]: Static tests should be in a different module, I think this is duplicated in all test classes

File: auto_remediate/test/test_securityhub_s3.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        # validate test
+        response = sh.client_s3.get_bucket_acl(Bucket="test")
+        assert response["Grants"][0]["Permission"] == "FULL_CONTROL"
+
+    def test_invalid_bucket(self, sh):
+        # create bucket
+        sh.client_s3.create_bucket(ACL="public-read", Bucket="test")
+
+        # validate test
+        assert not sh.s3_bucket_public_read_prohibited("test123")
```
Comment (jihygk) [inline auto_remediate/test/test_securityhub_s3.py:40]: Comment only: 
I think this gives a clue for how we should refactor the error handling - it should raise the NoSuchBucket exception (or whatever) directly and we should assert that that exception is raised and caught here.

That means we should have the try except in the lambda_handler remediate() function, I think

**Review by jihygk — CHANGES_REQUESTED** [review]
Tests look good except Static helper methods being duplicated, and the magic number for cmk backing key rotation test?

File: auto_remediate/test/test_securityhub_s3.py (hunk 3 above)
Comment (mlevit) [reply auto_remediate/test/test_securityhub_s3.py:40]: Doesn't that mean we'll be catching 20 different exceptions in the `lambda_handler`?

File: auto_remediate/test/test_securityhub_kms.py (hunk 2 above)
Comment (mlevit) [reply auto_remediate/test/test_securityhub_kms.py]: 7cf1cd0

**Pull request merged**